vxquery-core/src/test/java/org/apache/vxquery/datamodel/AbstractPointableTest.java

```diff
@@ -28,7 +28,9 @@
 
     protected void getTaggedValuePointable(Object value, IPointable result) throws IOException {
```

Comment:
I guess same applied for getValue method of ObjectPointable: it takes a pointable for the string without the tag